.azure-pipelines/azure-pipelines-image-template.yml

```diff
@@ -49,7 +49,11 @@ jobs:
           sudo modprobe overlay
           sudo apt-get install -y acl
           sudo bash -c "echo 1 > /proc/sys/vm/compact_memory"
-          ENABLE_DOCKER_BASE_PULL=y make PLATFORM=$(PLATFORM_AZP) PLATFORM_ARCH=$(PLATFORM_ARCH) $(BUILD_OPTIONS) configure
+          if [ ${PLATFORM_AZP} == marvell-arm64 ] || [ ${PLATFORM_AZP} == marvell-armhf ]; then
```

Comment:
Instead of the if-checks here, can the top-level pipeline build files be updated to point to `marvell` instead?

Reply:
@saiarcot895 If we look at the changes in the build commands, only configure command is different in "PLATFORM=marvell" string. Other commands are same. So I opted this change.
If we change name in top-level pipeline( ex: name: marvell-armhf) we need to change or checks multiple places such as how the build pipelines are fetched in "Azure.sonic-buildimage.official.marvell-armhf?branchName=master&label=Marvell-armhf" or while running a make command(make target/sonic-marvell-armhf.bin) for build. I thought this is less change, and does not break other arm64 platform builds.

Reply:
hmm, @xumia, thoughts?

Reply:
How about define a new parameter in azure-pipelines-build.yml for marvell-armhf and marvell-amd64.
https://github.com/sonic-net/sonic-buildimage/blob/7f454a778a3f324304c241ffcbb4fc012bf75c90/.azure-pipelines/azure-pipelines-build.yml#L100
For example:            PLATFORM: marvell

If the variable PLATFORM is not defined (empty), then use PLATFORM_AZP
```
[ -z "$PLATFORM" ] && PLATFORM=$(PLATFORM_AZP)
ENABLE_DOCKER_BASE_PULL=y make PLATFORM=$PLATFORM PLATFORM_ARCH=$(PLATFORM_ARCH) $(BUILD_OPTIONS) configure
// There is a little different $PLATFORM and $(PLATFORM), be careful.
```
Any other platforms have the similar requirements, only need to override the variable, no logic change.

Reply:
@xumia Have pushed the changes as recommended by you. Please review.